os_info.yaml

```diff
@@ -70,6 +76,17 @@ rhel-9:
     - aarch64
     - ppc64le
```

Comment:
s390x should be on rhel-9 as well as rhel8

Reply:
Added the s390x for rhel-9, and it was already added for rhel-8. However, I've noticed that the rhel-9 images did not make it to quay, so this needs to be fixed before merging this.

Reply:
Yep, I belatedly saw the rhel9 build problem. I'll see if I can fix that today.

Reply:
I have removed the rhel-9 entry from os-info in order to merge the PR so that we don't miss building the other OSs. Once it is fixed, we can easily add it.